docker/files/docker-dlc.sh

```diff
@@ -26,7 +26,12 @@ disable_mod()
 }
 
 # Enable or disable DLCs if configured
-if [[ ${DLC_SPACE_AGE:-} == "true" ]]; then
+if [[ -n "$MOD_LIST_MAP" ]]; then
+  # if MOD_LIST_MAP env variable is defined, then use it to determine the mods to enable
+  for mod in $MOD_LIST_MAP; do
+      ENABLE_MODS+=("$mod")
+  done
+elif [[ ${DLC_SPACE_AGE:-} == "true" ]]; then
```

Comment:
I would prefer to leave the current DLC logic untouched, and handle this before or after the dlc logic.
If it would be untouched, then the mention about the dlc mods must be included, can be removed because the existing logic will handle them correctly.